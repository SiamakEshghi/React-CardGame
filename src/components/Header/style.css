.header {
    background-color: forestgreen;
    display: flex;
    flex-direction: row;
    height: 80px;
    justify-content: space-around;
    align-items: center;
}
.logo {
    max-height: 60px;
    width: auto;
}