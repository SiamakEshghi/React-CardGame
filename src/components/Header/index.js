import React from 'react';
import logo from 'img/logo.png';
import 'components/Header/style.css';

export default () => {
    return (
        <div className="header">
            <img className="logo" src={logo}/>
            <h2>CARD GAME</h2>
            <h5>React App</h5>
        </div>
    );
}