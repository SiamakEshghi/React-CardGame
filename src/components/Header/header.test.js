import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from 'components/Header';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Header />);
});

describe('Header', () => {
    it('Should render correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});