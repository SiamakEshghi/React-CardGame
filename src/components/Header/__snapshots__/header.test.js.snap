// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render correctly 1`] = `
<div
  className="header"
>
  <img
    className="logo"
    src="logo.png"
  />
  <h2>
    CARD GAME
  </h2>
  <h5>
    React App
  </h5>
</div>
`;
